P/Pfapack/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Pfapack"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/xrq-phys/Pfapack.git", "0c71536b9641a0c8f4da67b373e3d4d5514561ab")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd Pfapack
+
+cmake fortran \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release
+make -j${nproc} VERBOSE=1
+make install
+
+# Manual conversion from static to dynamic lib.
+# Avoid linking libgfortran (these .f files make no reference to Fortran libs.)
+cc -shared -Wl,-force_load,${prefix}/lib/libpfapack.a \
+    -o ${libdir}/libpfapack.${dlext} \
+    -L${libdir} -lblastrampoline -lm \
+
+# Copy license file
+install_license LapackLicence
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+# No need to expand Fortran version since there code make no call to
+# standard Fortran libraries
+platforms = supported_platforms()
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libpfapack", :pfapack)
```

Comment:
The choice of the name of the variable is up to you, but for libraries we tend to call them `lib...`, just like the basename of the file.